steering_logic.py

```diff
@@ -3,25 +3,28 @@
 import numpy as np
 import math
 import pid
+from launch_utils import ControlMode, enter_control_mode
+from telemetry import KSPTelemetry
+
 
 # Consists of functions for steering and controlling a generic launch vehicle.
 # - Pitch Maneuver
 # - Roll Program
 # - MaxQ throttle
 # - constant acceleration
 
-def roll_program(mission_params, telem, vessel, conn, sc):
+def roll_program(mission_params, telem, vessel, conn, sc, telem_viz: KSPTelemetry):
```

Comment:
to facilitate refactor, should set this with default to None, so it doesnt break all the other avionics and the refactor can be done in a more controlled manner

Reply:
encapsulated this in `enter_contrl_mode` fn